examples/rust/external_data_loader/src/main.rs

```diff
@@ -56,18 +75,52 @@ fn main() -> anyhow::Result<()> {
 
     let rec = {
         let mut rec = rerun::RecordingStreamBuilder::new("rerun_example_external_data_loader");
-        if let Some(recording_id) = args.recording_id {
+        if let Some(recording_id) = args.recording_id.as_ref() {
             rec = rec.recording_id(recording_id);
         };
 
         // The most important part of this: log to standard output so the Rerun Viewer can ingest it!
         rec.stdout()?
     };
 
-    rec.log_timeless(
-        rerun::EntityPath::from_file_path(&args.filepath),
+    // TODO(#3841): really need those send APIs
```

Comment:
since this is example code to be consumed by users it would be nice(r) to write about what they can expect to improve here in the future inead :)